.travis.yml

```diff
@@ -34,45 +34,109 @@ env:
     # Default Elasticsearch heap size can be too large for Travis
     - ES_JAVA_OPTS="-Xms256m -Xmx512m"
 
-  matrix:
-    # sort modules by test time with quickest modules first
```

Comment:
You've lost this comment in your change; was that intentional? Would you like to bring it back?

Reply:
Sure. I'll add it back in.